components/logged_in_navigation/components/you/index.jade

```diff
@@ -7,7 +7,7 @@
     )
 
 .Dropdown.js-dropdown
-  .DropdownSection
+  .DropdownSection.DropdownSection--expandable.js-expandable( data-key='you', data-state='active' )
     a.DropdownItem.DropdownItem--primary( href="/#{user.get('slug')}" )
```

Comment:
Talked with @danpia in person, just add a link underneath this to the user profile, with the label `Profile`.